docs/users/extend/custom-commands.md

```diff
@@ -102,6 +102,16 @@ Useful variables for container scripts are:
 * DDEV_WEBSERVER_TYPE: nginx-fpm, apache-fpm
 * IS_DDEV_PROJECT: if set to "true" it means that php is running under DDEV
 
+### Annotations supported
+
+The custom commands support various annotations in the header which are used to provide additional information about the command to the user:
+
+* Description
+* Usage
+* Example (use `\n` to force a line break)
+* ProjectTypes
+* OSTypes
+
```

Comment:
This whole section will be changed again with #2493 . There I will provide an extended description of all annotations.